org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend

```diff
@@ -2390,11 +2429,22 @@ class ResolveTypeHierarchyItemParams {
 
 }
 
+@JsonRpcData
+class DocumentSymbolOptions implements WorkDoneProgressOptions {
```

Comment:
Many of the `*Options` implement `WorkDoneProgressOptions` and define the `Boolean workDoneProgress`. Would a super class make sense?